src/main/webapp/aboutus.html

```diff
@@ -23,6 +23,7 @@
       <ul id="navigation">
         <li><a href="/">Home</a></li>
         <li><a href="/aboutus.html">About Our Team</a></li>
+        <li><a href="/stats.html">Stats</a></li>
```

Comment:
We'll maybe want to consider moving this top navigational piece out into some kind of header file so that we don't have to update it everywhere, but this works for now!